samples/v1.2/Elements/ActionFallback.json

```diff
@@ -0,0 +1,150 @@
+{
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"type": "AdaptiveCard",
+	"version": "1.2",
+	"body": [
+		{
+			"type": "ActionSet",
+			"actions":[
+				{
+					"type": "NoExist",
+					"fallback": {
+						"type": "NeitherExist",
+						"fallback": 
+						{
```

Comment:
Nit: Can you edit the IDE settings to convert the tabs to spaces pls?

Reply:
actually, current policy for json is to use tabs :)

Reply:
Oh? I see - my bad then - any specific reason why? What are we using elsewhere?

---
In reply to: [271974436](https://github.com/Microsoft/AdaptiveCards/pull/2612#discussion_r271974436) [](ancestors = 271974436)

Reply:
We're using spaces everywhere else. For JSON, tabs are more accepted in the industry. Presumably because it's a data format where size is at more of a premium...

---
In reply to: [271975101](https://github.com/Microsoft/AdaptiveCards/pull/2612#discussion_r271975101) [](ancestors = 271975101,271974436)

Reply:
You can see our whitespace policies in the `.editorconfig` file at the root of the repo :)

---
In reply to: [271975957](https://github.com/Microsoft/AdaptiveCards/pull/2612#discussion_r271975957) [](ancestors = 271975957,271975101,271974436)

Reply:
Interesting - thanks for explaining!

---
In reply to: [271975957](https://github.com/Microsoft/AdaptiveCards/pull/2612#discussion_r271975957) [](ancestors = 271975957,271975101,271974436)